docs/source/api/proxy.md

```diff
@@ -1,22 +1,25 @@
-=======
-Proxies
-=======
+# Proxies
 
-Module: :mod:`jupyterhub.proxy`
-===============================
+## Module: {mod}`jupyterhub.proxy`
 
+```{eval-rst}
 .. automodule:: jupyterhub.proxy
+```
 
+```{eval-rst}
 .. currentmodule:: jupyterhub.proxy
+```
```

Comment:
Is the indentation off here?

The currentmodule directive was new to me as well. Is it like a "cd someplace" in a bash script to not need to give a full reference later?

Is it required to be in rst also? The auto... Needs to be in an eval-rst still though because they emit rST that needs to be parsed once more.

Reply:
Searched for this, its documented to be of relevance if we don't have automodule and have a standalone file and want to avoid writing a full path like `jupyterhub.app.JupyterHub`.

https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#directive-py-currentmodule

We have automodule, which results in py:module, so I've added a commit removing the `.. currentmodule` calls that were redundant. To have less magic we could go with full paths as well.